tensorboard/plugins/debugger/tf_debugger_dashboard/tf-session-runs-view.html

```diff
@@ -157,15 +156,23 @@
       this._clearTable();
       // TODO(cais): Support sorting.
       this._renderHeader();
+      let activeRow;
       for (const runKey in this._runKey2Count) {
         if (this._runKey2Count.hasOwnProperty(runKey)) {
           const sessionRun = JSON.parse(runKey);
           const count = this._runKey2Count[runKey];
           const isActive = this._activeRunKey === runKey;
           const numDevices = this._runKey2NumDevices[runKey];
-          this._renderRow(sessionRun, numDevices, count, isActive, this.soleActive);
+          const row = this._renderRow(sessionRun, numDevices, count, isActive, this.soleActive);
+          if (row) {
+            activeRow = row;
+          }
         }
       }
+
+      if (activeRow) {
+        activeRow.parentNode.parentNode.parentNode.scrollTop = activeRow.offsetTop;
```

Comment:
Could we use a more specific selector? In case one day, a future dev adds a container in between the layers?